system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -41,11 +41,7 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
    * @param topic the topic name used for publishing
    * @param publish_period the period at which metrics are published. 0 ms means don't publish
    */
-  LinuxCpuMeasurementNode(
-    const std::string & name,
-    const std::chrono::milliseconds measurement_period,
-    const std::string & topic,
-    const std::chrono::milliseconds publish_period);
+  LinuxCpuMeasurementNode(const std::string & name, const rclcpp::NodeOptions & options);
```

Comment:
The above doc isn't valid anymore. We could add expected node options and default behavior.

Reply:
Done.